source/classes.tex

```diff
@@ -4146,6 +4137,7 @@
 };
 \end{codeblock}
 \end{example}
+\end{note}
```

Comment:
I think the example is for the entire facility, not just for "need be defined only". So, please move the example out of the note.